im.fluffychat.Fluffychat.json

```diff
@@ -0,0 +1,71 @@
+{
+    "app-id": "im.fluffychat.Fluffychat",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "fluffychat",
+    "copy-icon": true,
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=fallback-x11",
```

Comment:
does flutter actually support wayland natively these days? if so, cool! if not, then this will break things for wayland users.

Reply:
https://github.com/flutter/flutter/pull/66519

Fluffychat added wayland support since the 0.21.0 version with https://gitlab.com/ChristianPauly/fluffychat-flutter/-/merge_requests/239
 
(and I can confirm that it works really well and without issues. X11 version was really blurry on my fractional scaling 4k)